packages/react-components/lib/simple-search.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { TextField, makeStyles, Divider } from '@material-ui/core';
+
+export interface OnChangeEvent {
+  name?: string | undefined;
+  value: string;
+}
+
+export interface SimpleSearchProps {
+  onChange?: (e: React.ChangeEvent<OnChangeEvent>) => void;
+  value: string;
+}
+
+const useStyles = makeStyles((theme) => ({
+  simpleSearch: {
+    margin: '1rem',
+  },
+  clearSearch: {
+    color: theme.palette.error.main,
+  },
+  searchBar: {
+    width: '100%',
+  },
+  divider: {
+    margin: '1.5rem 0',
+  },
+}));
+
+export const SimpleSearch = (props: SimpleSearchProps): JSX.Element => {
```

Comment:
Do you think changing "Search" to "Filter" across the PR makes it more user friendly? Technically it is filtering instead of searching, having a text input with "Filter" might be better UX.